.github/workflows/tests.yml

```diff
@@ -23,11 +23,10 @@ jobs:
       - uses: astral-sh/setup-uv@v6
         with:
           enable-cache: true
-      - run: echo "UV_PROJECT_ENVIRONMENT=$(python -c "import sysconfig; print(sysconfig.get_config_var('prefix'))")" >> $GITHUB_ENV
```

Comment:
This `UV_PROJECT_ENVIRONMENT` tricked `uv` into installing into system Python.

Using `activate-environment` from `setup-uv` eliminates the need for this complexity